packages/@okta/i18n/dist/properties/login.properties

```diff
@@ -484,6 +484,8 @@ oktaverify.sent = Push sent!
 oktaverify.rejected = You have chosen to reject this login.
 oktaverify.timeout = Your push notification has expired.
 oktaverify.warning = Haven't received a push notification yet? Try opening the Okta Verify App on your phone.
+oktaverify.numberchallenge.instruction = On your phone tap, <span class="challenge-number">{0}</span> on Okta Verify prompt to continue.
```

Comment:
there seems to be an extra comma in this sentence. It should be `On your phone tap ## on Okta Verify...`, or if you need a comma, it should be after phone `On your phone, tap ## on Okta Verify...`